app/pipedrive/views.py

```diff
@@ -41,4 +43,14 @@ async def callback(event: PipedriveEvent, tasks: BackgroundTasks):
         if company and company.tc2_agency_id:
             # We only update the client if the deal has a company with a tc2_agency_id
             tasks.add_task(update_client_from_company, company)
+    elif event.meta.object == 'activity':
+        pass
```

Comment:
If we don't want to accept webhooks for Activites, can we not just select that option in the Pipedrive webhook setup?

Reply:
This isn't done?